ci-operator/step-registry/baremetalds/assisted/setup/baremetalds-assisted-setup-commands.sh

```diff
@@ -114,7 +114,7 @@ set -e
 
 if [ "${JOB_TYPE:-}" = "presubmit" ] && (( ! \${IS_REHEARSAL} )); then
   # We would like to keep running a stable version for PRs
-  echo "export OPENSHIFT_VERSION=4.8" >> /root/config
+  echo "export OPENSHIFT_VERSION=4.9" >> /root/config
```

Comment:
Why do we need it also here? Assisted service define the default value.
I think we don't want to maintain the default values in multiple places (and repositories).

Reply:
I can try removing it
Probably a good idea